api/v1alpha1/olsconfig_types.go

```diff
@@ -110,6 +110,7 @@ type ConversationCacheSpec struct {
 // RedisSpec defines the desired state of Redis.
 type RedisSpec struct {
```

Comment:
We can have an option to choose whether the operator deploys a Redis server according to the configurations in CR.
like `operatorManaged: false|true`

Reply:
Sounds good to me. Will modify it accordingly.